Automattic/wp-calypso
# HelpSearch: remove duplicated <SearchCard /> component.

#### Changes proposed in this Pull Request

This PR modifies the Search card of the InlineHelp block in order to be reusable by the HelpSearch component which is located on the customer home page.
For this, I've added two new `placeholder` and `location` properties to the card search to be able to customize it.

#### Testing instructions

Once you apply these changes in your local dev env, check that both component work as expected. Confirm that it sets the location property properly when it records tracks events.
Pay attention to the placeholder value of the query search input element.

Fixes partially https://github.com/Automattic/wp-calypso/issues/42704

<img width="400" alt="Screen Shot 2020-06-17 at 4 27 05 PM" src="https://user-images.githubusercontent.com/77539/84941311-93005e00-b0b7-11ea-9e33-4396eecc614c.png">

<img width="400" alt="Screen Shot 2020-06-17 at 4 26 39 PM" src="https://user-images.githubusercontent.com/77539/84941321-972c7b80-b0b7-11ea-8edb-796bf07ab4ad.png">

<img width="400" alt="Screen Shot 2020-06-17 at 4 27 39 PM" src="https://user-images.githubusercontent.com/77539/84941294-8f6cd700-b0b7-11ea-98d8-d8059933a6cc.png">
